ocs_ci/framework/__init__.py

```diff
@@ -431,3 +431,8 @@ def switch_to_cluster_by_cluster_type(self, cluster_type, num_of_cluster=0):
 
 
 config = MultiClusterConfig()
+
+
+class globalVariables:
```

Comment:
also better to rename since it collides with python `global variables` which does add sense in naming

Reply:
The name was kept as globalvariable on purpose so that it feels similar and adds up to a quick understanding of the class. We are just using this variable TIMEREPORT_DICT to save data for processing, adding more methods to it will add complications when we have more different types of data added in this nested dictionary.

Reply:
ack